app/controllers/emails_controller.rb

```diff
@@ -7,7 +7,7 @@ def show
     @templates = EmailTemplate.with_status(get_status)
     @addresses = @event.email_addresses_of_type(get_status)
 
-    @email = Email.new(hide_recipients: true, reply_to: 'workshop.portal@hpi.de', recipients: @addresses,
+    @email = Email.new(hide_recipients: true, reply_to: 'workshop.portal@hpi.de', recipients: @addresses.join(','),
```

Comment:
I had to refactor the email_addresses_of_type(type) method, since it was returning a string with concatenated emails addresses in a random order on Travis. Only Travis knows why this wasn't an issue earlier, but tests were suddenly faling because of this. I changed it to return an array of Emails, which is better anyway, but made this change to join the addresses here necessary.